webdriver/tests/bidi/browsing_context/capture_screenshot/capture_screenshot.py

```diff
@@ -39,8 +39,8 @@ async def test_capture(bidi_session, top_context, inline, compare_png_bidi):
     assert comparison.equal()
 
 
-@pytest.mark.parametrize("delta_width", [-10, +20])
-@pytest.mark.parametrize("delta_height", [-30, +40])
+@pytest.mark.parametrize("delta_width", [-10, +20], ids=["width smaller than original", "width larger than original"])
```

Comment:
Note that this makes the testname quite long to something like: `test_capture_with_viewport[width smaller than original-height smaller than original", "height larger than original].

Maybe we can reduce to `ids=["width smaller", ...]`? I think that should be fine.

Reply:
Done